apps/serdes/Printer.h

```diff
@@ -0,0 +1,27 @@
+#ifndef HALIDE_PRINTER_H
+#define HALIDE_PRINTER_H
+
+#include <string>
+#include <vector>
+
+#include <Halide.h>
+using namespace Halide;
+
+class Printer {
```

Comment:
This class would be of much greater utility if you were able to pass it a `std::ostream&` instead of having it assume `stdout`.